content/posts/waishnav_deore_final_report.md

```diff
@@ -0,0 +1,183 @@
+
+Here is the video of search functionality which will be going to be used by CircuitVerse using [PgSearch](https://github.com/Casecommons/pg_search) gem.
+
+{{< video src="/images/waishnav_deore/topic-search.mp4" type="video/mp4" preload="auto" >}}
+
+---
+
+### 7. Leaderboard Ranking
+
+Ranking is important, if one need to judge the crediblity of the message. Using simple point distribution on forum thread creation, on comments etc etc, we can simply able to create a leaderboard of users.
+
+During the implementation, I got little confuse where the distribution of the points logic should exists, in the controller or in the model file. Then I read few blogs on fat model and slim controllers and vice versa.
+In the rails, these conventions are very helpful and important, depending on it's usecase one can utilize this concepts.
+
+Here is the screenshot of leaderboard page. Note: UI of leaderboard may changes later
+
+![leaderboard-ranking](/images/waishnav_deore/leaderboard-ranking.png)
+
+---
+## Pull Requests
+
+- `Week 1:` [Adding supported Rails versions in Appraisal](https://github.com/CircuitVerse/simple_discussion/pull/22)
+- `Week 2:` [Fixing CI fails](https://github.com/CircuitVerse/simple_discussion/pull/23)
+- `Week 3:` [Adding language filters and UI changes](https://github.com/CircuitVerse/simple_discussion/pull/24)
+- `Week 4:` [Report as spam and moderation review page](https://github.com/CircuitVerse/simple_discussion/pull/25)
+- `Week 5:` [Slack notification in specific channel for new thread](https://github.com/CircuitVerse/CircuitVerse/pull/5005)
+- `Week 6:` [Markdown editor using SimpleMDE](https://github.com/CircuitVerse/simple_discussion/pull/26)
+- `Week 7:` [Markdown syntax extension](https://github.com/CircuitVerse/simple_discussion/pull/27)
+- `Week 8:` [Topic search on Forum Threads](https://github.com/CircuitVerse/simple_discussion/pull/28)
+- `Week 9:` [Moderator delete access](https://github.com/CircuitVerse/simple_discussion/pull/29)
+- `Week 10:` [Leaderboard Page and Ranking](https://github.com/CircuitVerse/simple_discussion/pull/32)
+- `Week 11:` [Improved Documentation](https://github.com/CircuitVerse/simple_discussion/pull/33)
+
+---
+
+## Blogs
+CV community encourage to document each and every aspect of project. So I posted the biweely blogs and shared the progress as well as the implementation details in depth of each features and documented the whole project.
+You guys can checkout these blogs [here](https://waishnav.github.io/blog/tags/gsoc/)
+
+
+## What's Next?
+
+I’m excited to continue to contribute to this incredible open-source project which helps students to design and learn about digital circuit design.
+
+Additionaly, I started to work on this project with the thought that this revamped forum gemd will be distributed and published as different name, instead of `simple_discussion`
```

Comment:
```suggestion
Additionaly, I started to work on this project with the thought that this revamped forum gem will be distributed and published as different name, instead of `simple_discussion`
```